rpc/rest/security/rest_security.go

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate protoc --proto_path=model/http-security --gogo_out=model/http-security model/http_security/httpsecurity.proto
+
+package security
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/gorilla/mux"
+	"github.com/ligato/cn-infra/logging"
+	httpsecurity "github.com/ligato/cn-infra/rpc/rest/security/model/http-security"
+	"github.com/pkg/errors"
+	"github.com/unrolled/render"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	// Helps to obtain authorization header matching the field in a request
+	authHeaderStr = "authorization"
+	// Used to sign the token // todo maybe use something more secure
+	secret = "secret"
```

Comment:
This needs to be loaded from config or env, otherwise anyone can simply guess or read it from the source and easily manipulate token data.